CHANGES/936.contrib.rst

```diff
@@ -0,0 +1,2 @@
+Add test coverage for _multidict_base.py 
+`NotImplemented`` and `Iterable-but-not-Set` for `and, or, sub, xor` operators
```

Comment:
This should clear the Sphinx errors, plus brush up the style a bit. With full sentences et al.
```suggestion
Added test coverage for the :ref:`and <python:and>`, :ref:`or <python:or>`, :ref:`sub <python:sub>`, and :ref:`xor <python:xor>` operators in the :file:`multidict/_multidict_base.py` module. It also covers  
:py:data:`NotImplemented` and ":py:class:`~typing.Iterable`-but-not-:py:class:`~Set`" cases there.
```